src/NuGet.Core/NuGet.Protocol/Strings.resx

```diff
@@ -333,6 +342,10 @@ The "ms" should be localized to the abbreviation for milliseconds.</comment>
     <value>The server responded with HTTP '403 Forbidden' when accessing the source '{0}'. This suggests that the server has authenticated your identity but has not permitted you to access the requested resource. Provide credentials that have permissions to view this resource.</value>
     <comment>{0} is the source that needs credentials to be accessed.</comment>
   </data>
+  <data name="Http_UrlNotFound" xml:space="preserve">
+    <value>The server responded with HTTP '404 Not Found' when accessing the source '{0}'. This suggests that the server reported that the URL requested does not exist. Change the URL to a resource that exists.</value>
```

Comment:
What is the purpose of this explanation? 404 is a widely used error, right? Are you trying to tell the user that the server may also return 404 for reasons related permissions issues?

If that's your intention, then I don't like the instruction about changing "to a resource that exists" because it's both obvious in the 'not exist' case, and inaccurate in the 'no permissions' case. Can we just omit it?

"This suggests" is a little wishy washy and I'm not sure users will understand that we're saying "one of the reasons is that" the URL doesn't exist.

Reply:
following the pattern of the existing messages. let me think.

Reply:
got rid of that 2nd sentence in next iteration all together. yes, we could be more prescriptive.

Reply:
This was marked resolved, but I don't see any changes here?